### PR TITLE
BAVL-10 initial first commit for replatforming of the old book a video link to the new environment.

### PR DESCRIPTION
**Pull request by matt-crowson-moj**
This PR sets up a new development namespace for the re-platforming of the **book a video link** project.

There is another similar (older) namespace for **book a video link**.  However we will be replacing that one in due course.

### CODE REVIEW

File: namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-book-a-video-link-dev/00-namespace.yaml [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  labels:
+    cloud-platform.justice.gov.uk/is-production: "false"
+    cloud-platform.justice.gov.uk/environment-name: "development"
+    pod-security.kubernetes.io/enforce: restricted
+  annotations:
+    cloud-platform.justice.gov.uk/business-unit: "HMPPS"
+    cloud-platform.justice.gov.uk/slack-channel: "farsight-devs"
+    cloud-platform.justice.gov.uk/application: "Book a video link"
+    cloud-platform.justice.gov.uk/owner: "farsight-devs: dps-hmpps@digital.justice.gov.uk"
+    cloud-platform.justice.gov.uk/source-code: "https://github.com/ministryofjustice/hmpps-book-a-video-link.git,https://github.com/ministryofjustice/hmpps-book-a-video-link-api.git"
```
Comment (matt-crowson-moj) [inline namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-book-a-video-link-dev/00-namespace.yaml:14]: @timharrison-moj are we happy we these Git repo names?

Comment (timharrison-moj) [reply namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-book-a-video-link-dev/00-namespace.yaml:14]: Yes, look fine to me. Very much like the original, with an added 'a'
The thing to bear in mind - we want to deploy this under a different name / cert - alongside the original - and then switch the DNS to rollout.

File: namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-book-a-video-link-dev/resources/rds-postgresql.tf [hunk 2]
```diff
@@ -0,0 +1,146 @@
...
+  is_production          = var.is_production
+  namespace              = var.namespace
+  team_name              = var.team_name
+
+  # If any other inputs of the RDS is passed in the source db which are different from defaults,
+  # add them to the replica
+
+  # PostgreSQL specifics
+  db_engine         = "postgres"
+  db_engine_version = "16"
```
Comment (timharrison-moj) [inline namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-book-a-video-link-dev/resources/rds-postgresql.tf:62]: Don't think we need a read-replica?

File: namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-book-a-video-link-dev/resources/rds-postgresql.tf [hunk 3]
```diff
@@ -0,0 +1,146 @@
...
+
+resource "kubernetes_secret" "read_replica" {
+  # default off
+  count = 0
+
+  metadata {
+    name      = "rds-postgresql-read-replica-output"
+    namespace = var.namespace
+  }
+
```
Comment (timharrison-moj) [inline namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-book-a-video-link-dev/resources/rds-postgresql.tf:119]: Don't need?

**Review by timharrison-moj — DISMISSED** [review]
Minor comments/questions only.
LGTM for the rest.

File: namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-book-a-video-link-dev/resources/rds-postgresql.tf (hunk 2 above)
Comment (matt-crowson-moj) [reply namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-book-a-video-link-dev/resources/rds-postgresql.tf:62]: Yeah I wasn't sure this came about as a result of using the cloud platform CLI tool to generate this.

File: namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-book-a-video-link-dev/resources/rds-postgresql.tf (hunk 3 above)
Comment (matt-crowson-moj) [reply namespaces/live.cloud-platform.service.justice.gov.uk/hmpps-book-a-video-link-dev/resources/rds-postgresql.tf]: Removed.

**Review by FolarinOyenuga — APPROVED** [review]

**Pull request merged**